.github/workflows/python-build.yml

```diff
@@ -88,4 +96,5 @@ jobs:
     - name: Check python package
       run: twine check dist/*
     - name: Publish package distributions to PyPI
+      if: github.event_name == 'release' && github.event.action == 'published'
```

Comment:
In the issue I filed yesterday, I outline that the entire job has to be remade, split into parts with privileges scoped + the job conditional is broken. Once that's done, this clause would only be necessary in the job level.

Reply:
Putting a * in front of a PR link makes it more readable..
* #915

Reply:
I have reverted all but one change to this file to leave room for #915 to do it correctly.